docker-init-script/src/main/java/eu/xenit/docker/alfresco/InitScriptMain.java

```diff
@@ -134,24 +134,8 @@ public void process() {
             globalProperties.put("alfresco-pdf-renderer.exe", "${alfresco-pdf-renderer.root}/alfresco-pdf-renderer");
         }
 
-        javaOptions.add("-Xms" + environment.get("JAVA_XMS"));
-        javaOptions.add("-Xmx" + environment.get("JAVA_XMX"));
         javaOptions.add("-Dfile.encoding=UTF-8");
 
-        if ("true".equalsIgnoreCase(environment.get("JMX_ENABLED"))) {
-            javaOptions.add("-Dcom.sun.management.jmxremote.authenticate=false");
-            javaOptions.add("-Dcom.sun.management.jmxremote.local.only=false");
-            javaOptions.add("-Dcom.sun.management.jmxremote.ssl=false");
-            javaOptions.add("-Dcom.sun.management.jmxremote");
-            javaOptions.add("-Dcom.sun.management.jmxremote.rmi.port=5000");
-            javaOptions.add("-Dcom.sun.management.jmxremote.port=5000");
-            javaOptions.add("-Djava.rmi.server.hostname=" + environment.get("JMX_RMI_HOST"));
-            globalProperties.put("alfresco.jmx.connector.enabled", "true");
```

Comment:
I think we will still need this property to be set?

Reply:
See slack discussion.
Works fine without setting it.

Reply:
> Toon Geens 12:00
> I'm still wondering what the alfresco.jmx.connector.enabled option does
> see AlfrescoRmiRegistryFactoryBean
> that's just Alfresco/Spring itself managing JMX in code ?
> 
> Roxana Angheluta 13:36
> I just tried with a container having alfresco.jmx.connector.enabled=false and connecting via jmx works fine on port 5000 without authentication

Reply:
Okay then